tests/unit/RecipientsTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace PagarMe\Test;
+
+use PagarMe\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\Psr7\Response;
+use PagarMe\Endpoints\Customers;
+use PagarMe\Endpoints\Endpoint;
+
+final class RecipientTest extends PagarMeTestCase
+{
+    public function recipientProvider()
+    {
+        return [
+            [
+                new MockHandler([
+                    new Response(200, [], self::jsonMock('RecipientMock'))
+                ])
+            ]
+        ];
+    }
+
+    public function balanceProvider()
+    {
+        return [
+            [
+                new MockHandler([
+                    new Response(200, [], self::jsonMock('BalanceMock'))
+                ])
+            ]
+        ];
+    }
+
+    /**
+     * @dataProvider recipientProvider
+     */
+    public function testRecipientCreate($mock)
+    {
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->recipients()->create([
```

Comment:
It's good practice to use single quotes when there is no need for string interpolation :wink: